AnimatedTextInput/Classes/AnimatedTextInput.swift

```diff
@@ -93,6 +93,7 @@ public class AnimatedTextInput: UIControl {
     // MARK: Configuration
 
     private func addLineViewConstraints() {
+        self.removeConstraints(self.constraints)
         pinLeading(toLeadingOf: lineView, constant: style.leftMargin)
```

Comment:
Nit: For style consistency with the rest of the code, I'd remove`self` in this case.